pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1632,6 +1632,21 @@ let
         };
       };
 
+      kalebpace.balena-vscode = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "balena-vscode";
+          publisher = "kalebpace";
+          version = "0.1.3";
+          sha256 = "sha256-CecEv19nEtnMe0KlCMNBM9ZAjbAVgPNUcZ6cBxHw44M=";
+        };
+        meta = with lib; {
+          description = "Unofficial VS Code extension for use with Balena Cloud";
```

Comment:
```suggestion
          description = "A VS Code extension for integration with Balena";
```

Reply:
`changelog` and `downloadPage` are missing in meta.
Please copy from other examples in file as reference.